### PR TITLE
Update all_actions.py

### PR DESCRIPTION
**Pull request by envirodug**
See https://github.com/Amber-MD/pytraj/issues/1397

### CODE REVIEW

**Comment by hainm** [comment]
@envirodug  you don't need to make 3 separate branches for this. 
Just update changes in a single one since they are related. 

Example: https://github.com/Amber-MD/pytraj/pull/1395/files

**Comment by hainm** [comment]
http://amber-md.github.io/pytraj/latest/developer_guide.html#our-github-repo

**Comment by envirodug** [comment]
OK, thanks for the links on how to update the code on github. This is my first time ... if you can't tell.

**Comment by hainm** [comment]
> This is my first time ... if you can't tell.

no worry. It's fun to learn new thing.

**Comment by envirodug** [comment]
Hmmmm, so making the change to `command = ' '.join((mask,options))` in `def atomicfluct` does not seem to fix the problem.  ¯\_(ツ)_/¯

**Comment by hainm** [comment]
you meant that change won't pass your local test?

**Comment by envirodug** [comment]
Correct.  It fails when I run `pytest -vs test_analysis/test_atomicfluct.py`. Maybe it would be good if you could test it, to make sure I am not doing something wrong.

**Comment by hainm** [comment]
can you push your updated test to this PR so I can fork from yours and make change?

**Comment by envirodug** [comment]
I edited the previously file on the github website, not locally, and I'm having trouble figuring out how to add a new file to an existing pull request.

**Comment by hainm** [comment]
```bash
git clone https://github.com/envirodug/pytraj # your git repo
cd pytraj
git checkout patch-1 # this is the branch you make pull request here.
# look at your 1st post:
# "envirodug  wants to merge 2 commits into Amber-MD:master from envirodug:patch-1"

# add changes to tests/test_analysis/test_atomicfluct.py

# commit your change
git add tests/test_analysis/test_atomicfluct.py
git commit -m "add test for test_atomicfluct.py"
git push origin patch-1 # update your change to the pull requested branch. 
# that's it. That change will be updated in this PR.
```

**Comment by envirodug** [comment]
OK I think I did it. Thanks again for the steps.

**Comment by hainm** [comment]
yeah. So now we can see how your change will be built and tested here: https://travis-ci.org/Amber-MD/pytraj/builds/252175136

File: tests/test_analysis/test_atomicfluct.py [hunk 1]
```diff
@@ -40,6 +40,16 @@ def test_RMSF(self):
         data = pt.rmsf(t0)
         aa_eq(data, state.data[-1].values)
 
+    def test_RMSF(self):
```
Comment (hainm) [inline tests/test_analysis/test_atomicfluct.py]: @envirodug you need to update the method name. There is already a 'test_RMSF'.

may be 'test_RMSD_with_options'?

**Comment by envirodug** [comment]
I changed it to `def test_RMSF_with_options()` and still had the same problem.

**Comment by hainm** [comment]
from travis log
```bash
test_analysis/test_atomicfluct.py::TestAtomicFluct::test_RMSF_with_options PASSED
```

https://travis-ci.org/Amber-MD/pytraj/jobs/252184407

PS: Have your "python setup.py install" again after making changes? :P

**Comment by hainm** [comment]
Your change works for me. Merge now. Thanks for your 1st contribution ;).

```python
In [3]: import pytraj as pt

In [4]: traj = pt.iterload('tz2.nc', top='tz2.parm7')

In [5]: pt.rmsf(traj, '@CA', options='')
Out[5]: 
array([[   5.        ,    4.70036915],
       [  16.        ,    3.74434106],
       [  40.        ,    2.48968198],
       ..., 
       [ 160.        ,    2.33881004],
       [ 174.        ,    3.27343503],
       [ 198.        ,    4.13646535]])

In [6]: pt.rmsf(traj, '@CA', options='byres')
Out[6]: 
array([[  1.        ,   4.70036915],
       [  2.        ,   3.74434106],
       [  3.        ,   2.48968198],
       ..., 
       [ 10.        ,   2.33881004],
       [ 11.        ,   3.27343503],
       [ 12.        ,   4.13646535]])
```

**Comment by envirodug** [comment]
Yes I forgot to run setup/install. Rookie mistake.

**Pull request merged**